doc/submitting.md

```diff
@@ -0,0 +1,119 @@
+# Submitting
+
+The submission guide for replacement models for AoE1, AoE2, SWGB and the Definitive Editions.
+
+## Software
+
+**3D Assets**
+
+* [Blender](https://www.blender.org/) (version >= 2.8): 3D modeling, animating and rendering
+
+Using Blender is required. We currently do no accept other formats.
+
+**2D Assets**
+
+Recommended:
+
+* [Krita](https://krita.org/): great out-of-the-box image manipulation and drawing tool
+* [GIMP](https://www.gimp.org/): swiss army knife of image editing with loads of plugins and extensions available
+
+Photoshop, Corel or Paint are also allowed, although we prefer open formats as well as software that work on GNU/Linux and BSD.
```

Comment:
If we mention these, could we please also mention Affinity Photo as this is the alternative (also used in industry works) to Photoshop.